jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/codeAction/proposal/quickfix/InsertAnnotationQuickFix.java

```diff
@@ -31,20 +33,21 @@
 import org.eclipse.lsp4jakarta.jdt.codeAction.proposal.ModifyAnnotationProposal;
 
 /**
- * Quickfix for adding missing attributes to annotations
+ * Quickfix for adding missing attributes to new or existing annotations
```

Comment:
Can we clarify a bit further?
"Quickfix for adding new annotations or adding attributes to existing annotations"

Reply:
will do that